types/artifacts.d.ts

```diff
@@ -63,6 +63,8 @@ declare global {
       ConsoleMessages: Crdp.Log.EntryAddedEvent[];
       /** All the iframe elements in the page.*/
       IFrameElements: Artifacts.IFrameElement[];
+      /** The contents of the initial HTML document request. */
+      InitialHtml: string;
```

Comment:
```suggestion
      MainDocumentContent: string;
```

📣  Any objections?

Reply:
main-document wfm and `MainDocumentContent` sgtm if we can align on them elsewhere :)